### DISCUSSION TITLE
Anyone got any idea on how to make this go online?

### DISCUSSION THREAD

**Discussion by ChrisYOLO** [easytarget/esp32-cam-webserver#243] [Q&A]
I am preety new to this and i dont know much about networking and arduino. I got this working and i tweaked the settings a bit so i could hopefully get some awnsers to my problem. Nothing.. I need to add a camera to my store so i can see it when i am at my house for launch and i thought this would be a nice little project of mine. Because using this camera as a local Ip camera has no much use to me, anyone got any idea on how to make this go online? I have changed the port on my config tab and set a stable ip on both esp and router ip reservation, i also port forwared and opened the right port but i still couldn't connect to the camera. At this point i dont know much so i cant really move one :/ If anyone is willing to help me i would really appreciate it :D

**Comment by sonofcy** [comment]
It sounds like you are on the right track BUT is your security ok? Why not simply buy an off the shelf camera for your business. They are inexpensive and since it's for your business tax deductible so cheaper still. You can make this work, but it's very technical and if not done correctly a major security exposure. Now worth it in my opinion, spend a few bucks for a Nest or Arlo. Another advantage to a store bought solution is you can save videos in cloud for some length of time in case you discover something like inventory walking out the door.

**Comment by easytarget** [comment]
Sorry; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
